client/executor/wasmi/src/lib.rs

```diff
@@ -295,9 +319,19 @@ impl<'a> wasmi::ModuleImportResolver for Resolver<'a> {
 			}
 		}
 
-		Err(wasmi::Error::Instantiation(
-			format!("Export {} not found", name),
-		))
+		if self.enable_stub {
+			trace!("Could not find function {}, a stub will be provided instead.", name);
```

Comment:
Let's add `target` here. Confusingly, it is `target: "wasm-executor"` in this module.

Reply:
Can you provide me the exact message you want? I'm new :pray:

Reply:
The message is actually good! I mean something like

```suggestion
			trace!(target: "wasm-executor", "Could not find function {}, a stub will be provided instead.", name);
```

This way it would allow us to filter messages with `RUST_LOG` that come only from this module.